package.json

```diff
@@ -35,7 +35,6 @@
         "husky": "8.0.3",
         "js-yaml": "4.1.0",
         "lint-staged": "15.2.9",
-        "nango": "^0.44.0",
```

Comment:
Any reason to remove?

Reply:
Oh, that was just a mistake on my part. I was having issues with overriding to my local install and I think I deleted it to fix, not meaning to commit it.

Reply:
Fixed in e8fba64